src/bin/pgcopydb/copydb.c

```diff
@@ -709,14 +709,9 @@ copydb_init_table_specs(CopyTableDataSpec *tableSpecs,
 		.cfPaths = &(specs->cfPaths),
 		.pgPaths = &(specs->pgPaths),
 
-		.source_pguri = { 0 },
-		.target_pguri = { 0 },
-		.sourceSnapshot = {
-			.pgsql = { 0 },
-			.pguri = { 0 },
-			.connectionType = specs->sourceSnapshot.connectionType,
-			.snapshot = { 0 }
-		},
+		.source_pguri = specs->source_pguri,
+		.target_pguri = specs->target_pguri,
```

Comment:
Please use the `&(...)` notation for the pointer assignment, like we do in the rest of the code.

Reply:
Done